Polyhedron/demo/Polyhedron/Scene_plane_item.h

```diff
@@ -28,7 +28,31 @@ class SCENE_BASIC_OBJECTS_EXPORT Scene_plane_item
   ~Scene_plane_item();
 
   double scene_diag() const {
-    const Scene_item::Bbox& bbox = scene->bbox();
+    /*If no item is visible, scene->bbox is 0,0,0,0,0,0 and the texture is empty.
+    To avoid that, we need to compute the scene's bbox if the items were visible.
+    {
+*/
+    Scene_item::Bbox bbox = scene->bbox();
+    if(bbox == Scene_item::Bbox(std::numeric_limits<double>::infinity(),
+                                std::numeric_limits<double>::infinity(),
+                                std::numeric_limits<double>::infinity(),
+                                -std::numeric_limits<double>::infinity(),
+                                -std::numeric_limits<double>::infinity(),
+                                -std::numeric_limits<double>::infinity()))
+      bbox = Scene_item::Bbox(0,0,0,0,0,0);
+    if(bbox == Scene_item::Bbox(0,0,0,0,0,0))
+    {
+      for(int id = 0; id< scene->numberOfEntries(); ++id)
+      {
+        if(scene->item(id)->isFinite() && !scene->item(id)->isEmpty())
+          bbox = bbox + scene->item(id)->bbox();
+      }
+    }
+    else
+    {
+      bbox = scene->bbox();
+    }
+    //}
```

Comment:
If all the scene items are hidden, the scene's Bbox is as described in the comment. The size of a plane_item depends on the scene's Bbox, so if a plane is recomputed with all items hidden, it will have all its vertices in (0,0,0). This code avoids that behavior by computing the the bbox _as_ if the items were displayed. (the as is missing in the comment)